cads_adaptors/adaptors/multi.py

```diff
@@ -24,9 +25,12 @@ def split_request(
             # get request values for that key
             req_vals = full_request.get(key, [])
             # filter for values relevant to this_adaptor:
-            these_vals = [
-                v for v in ensure_list(req_vals) if v in this_values.get(key, [])
-            ]
+            if key in dont_split_keys:
+                these_vals = req_vals
```

Comment:
should we ensure_list here?

Reply:
Fixed in 655f86a

Reply:
that is not what I meant :) Shouldn't it be `these_vals = ensure_list(req_vals)`?

Reply:
not necessarily, the point is we don't touch them, a string would be a valid option, and I am happy not to touch them